pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -100,10 +99,8 @@ func asCamelCase(key string) string {
 
 func generateRegistry(t *testing.T) string {
 	tmpl, err := template.New("fn").Parse(`
-// Is{{.CamleCase}}Enabled checks for the flag: {{.Flag.Name}}{{.Ext}}
-func (ft *FeatureToggles) Is{{.CamleCase}}Enabled() bool {
-	return ft.manager.IsEnabled("{{.Flag.Name}}")
-}
+{{"\t"}}// Flag{{.CamleCase}} {{.Flag.Name}}{{.Ext}}
```

Comment:
Was CamleCase a typo or intentional?
```suggestion
{{"\t"}}// Flag{{.CamelCase}} {{.Flag.Name}}{{.Ext}}
```

Reply:
you are too kind :facepalm: